README.md

```diff
@@ -310,19 +339,45 @@ Total reclaimed space: 1.614GB
 
 If you encounter any issues, please contact us on [discord](https://discord.com/invite/powerloom).
 
+## 4\. Case studies
 
+### 4.1 Deploy subsets of slots with different configs
+
+Meet Bob, The Architect of Chaos. Bob has five Powerloom slots, but he’s got an itch to make things complicated—deploying three nodes each with a different RPC and two nodes sharing a single RPC. Here’s how he pulls it off:
+
+1. Bob logs into his VPS and follows [Preparation](#1-preparation).
+2. When initializing the environment (by `./bootstrap.sh`), he inputs all the necessary slot info and sets **RPC1**.
+3. Bob then runs the deploy script: (by `python multi_clone.py`). Since he’s deploying a subset of slots, he follows [Deploy a subset of slots](#221-deploy-a-subset-of-slots) and sets the **start slot** and **end slot** to **slot1**. BOOM! **Slot1 is live.**
+4. Bob repeats the process for **slot2** using **RPC2** and deploys **slot2** using the same method.
+5. He does the same for **RPC3** to deploy **slot3**.
+6. Now for the final trick, Bob wants **slot4 and slot5** to share **RPC4**. He reinitializes the env variables by (by `./bootstrap.sh`), and when deploying(by python multi\_clone.py), he sets **start slot to slot4 and end slot to slot5**.
+
+BOOM! Bob did it. **Three slots running on three different RPCs and two slots sharing a single RPC.** Mission accomplished. 🎉
+
+### 4.2 Running Slots from Different Wallets on a Single VPS
+
+Meet Alice, the queen of efficiency. Unlike Bob, who enjoys juggling multiple RPCs, Alice has a different challenge. Her 3 Powerloom slots are scattered across 3 wallets, but she wants them all running smoothly on a single VPS. Here’s how Alice pulls it off:
```

Comment:
I suggest cloning the multiple_clone set up in 3 directories (one for each wallet) and using that instead.